Tools/jit/schema.py

```diff
@@ -0,0 +1,171 @@
+# pylint: disable = missing-class-docstring
+
+"""Schema for the JSON produced by llvm-readobj --elf-output-style=JSON."""
```

Comment:
Hmm, seems like depending on llvm-readobj/JSON here is a bad form. Why not use libelf directly instead of writing this in python?

Reply:
We also need to parse COFF (on Windows) and Mach-O (on macOS). `llvm-readobj` can handle all three, and the JSON output mostly works for all three (despite the command-line switch's name). It would probably make sense to push LLVM to make COFF/Mach-O JSON serialization officially supported, so this is less fragile.

Plus, keeping things in Python is a Good Thing around here. :)